.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
This repository uses [minitest](https://github.com/minitest/minitest), rather than [RSpec](https://rspec.info/) as the testing framework.

Would you mind deleting the `.rspec` file and the `spec/spec_helper.rb` file?

If you're having trouble running the tests locally, you can do so by:
1. Installing the bundle 
```bash
$ bundle install
```
2. Using Rake to run the tests 
```bash
$ bundle exec rake tests
```

If this doesn't work, let us know and we can try to troubleshoot.

Reply:
Alright